scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+open_browser_and_login() {
+    echo "Logging you in via $1 ..."
+    platform="$(uname)"
+    if [[ "$platform" == "Linux" ]]; then
+        xdg-open "$1" >/dev/null &
+    elif [[ "$platform" == "Darwin" ]]; then
+        open "$1" &
+    else
+        echo "Unsupported platform '$platform', please open $1 in a browser" 1>&2
+    fi
+}
+
+login_if_no_auth_error() {
+    auth_error="$(sed -E '/^.*#error=([^&]+)(&.*)?$/!d;s//\1/' <<<"$target_url" | tr '+' ' ')"
+    if [[ -z "$auth_error" ]]; then
+        open_browser_and_login $1
```

Comment:
Please use quoting around substitutions because bash may freak out on some special characters.
```suggestion
        open_browser_and_login "$1"
```

Reply:
Good to know, what inputs would that be? If $1 contains characters like " ?

Reply:
I don't remember exactly but I observed that bash doesn't like complicated URLs. Possibly with arguments, e.g. `&`, `=` and `?`.